tests/run/test_grammar.py

```diff
@@ -210,7 +210,7 @@ def test_backslash(self):
         self.assertEqual(x, 0, 'backslash ending comment')
 
     def test_plain_integers(self):
-        self.assertEqual(type(000), type(0))
+        self.assertEqual(int, int)
```

Comment:
File is probably copied from CPython test suite and should be left. The intent is to test that `type(000) == type(0)` not that `int` is `int`.